reactnative/RNPlayground/app/navigators/AppNavigator.tsx

```diff
@@ -33,36 +32,33 @@ import { navigationRef, useBackButtonHandler } from "./navigationUtilities"
  *   https://reactnavigation.org/docs/typescript#type-checking-the-navigator
  *   https://reactnavigation.org/docs/typescript/#organizing-types
  */
-export type AppStackParamList = {
-  Welcome: undefined
+export type NavigatorParamList = {
+  Welcome: undefined,
+  About: undefined,
+  Playground: undefined,
   // 🔥 Your screens go here
 }
 
+const Tab = createMaterialBottomTabNavigator<NavigatorParamList>()
+
+function AppTabs() {
```

Comment:
Uppercased functions? Is this a RN thing?

Reply:
Yes, more often than not, RN uses a `PascalCase` naming convention for funcs - despite having no official style guide